bot/exts/moderation/infraction/_scheduler.py

```diff
@@ -162,20 +162,12 @@ async def apply_infraction(
         # apply kick/ban infractions first, this would mean that we'd make it
         # impossible for us to deliver a DM. See python-discord/bot#982.
         if not infraction["hidden"]:
-            dm_result = f"{constants.Emojis.failmail} "
-            dm_log_text = "\nDM: **Failed**"
-
-            # Sometimes user is a discord.Object; make it a proper user.
-            try:
-                if not isinstance(user, (discord.Member, discord.User)):
-                    user = await self.bot.fetch_user(user.id)
-            except discord.HTTPException as e:
-                log.error(f"Failed to DM {user.id}: could not fetch user (status {e.status})")
+            if await _utils.notify_infraction(infraction, user, user_reason):
+                dm_result = ":incoming_envelope: "
```

Comment:
Should this emoji perhaps be put in `constants.Emoji`?

Reply:
I don't know. It's a standard Unicode emoji, which normally doesn't have a constant. The only one that does is `ok_hand`, but the constant is only used once out of the nine references to that emoji.

Reply:
I just noticed the emojis right above `ok_hand` in the config. Didn't recognise them at first since they were Unicode escapes. Still, it's not very consistent, and I imagine it being cumbersome to use a constant instead of e.g. just typing `":x:"`.